src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -195,8 +193,6 @@ Executing this command will generate the queries, mutations, and subscriptions w
 
```

Comment:
Same here with this sentence abive - can we expand on this - why am I doing this step here vs later with the next section? Can add detail on why it is needed in this order?